mlora/lora_liner.py

```diff
@@ -29,11 +29,11 @@ def set_parameter(self, r: int, alpha: int, dropout: float):
         self.scaling_ = alpha / r
 
     def forward(self, data: torch.Tensor) -> torch.Tensor:
-        data_ = F.dropout(data, self.dropout_)
+        data_ = F.dropout(data.type_as(self.lora_a_), self.dropout_)
```

Comment:
use to function, dose type_as is in-place op???

Reply:
<img width="716" alt="image" src="https://github.com/TUDB-Labs/multi-lora-fine-tune/assets/16332179/037f1e3c-f1ce-44a8-a852-10573421d7d9">

Reply:
https://discuss.pytorch.org/t/why-does-pytorch-needs-the-three-functions-to-type-and-type-as/20164/4 use this